test/ios-specs.js

```diff
@@ -3,7 +3,7 @@
 import { fixes, XcodeCheck, XcodeCmdLineToolsCheck, DevToolsSecurityCheck,
          AuthorizationDbCheck, CarthageCheck, OptionalApplesimutilsCommandCheck,
          OptionalIdbCommandCheck, OptionalIdevicelocationCommandCheck,
-         OptionalIOSDeployCommandCheck, OptionalIOSWebkitDebugProxyCommandCheck, OptionalIfuseCommandCheck } from '../lib/ios';
+         OptionalIOSWebkitDebugProxyCommandCheck } from '../lib/ios';
```

Comment:
OptionalIOSWebkitDebugProxyCommandCheck and OptionalIdevicelocationCommandCheck can be removed as well